examples/pytorch/pytorch_example_script_multigpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#SBATCH --job-name=pytorch_mnist     # job name
+#SBATCH --ntasks=1                   # number of MP tasks
+#SBATCH --ntasks-per-node=1          # number of MPI tasks per node
+#SBATCH --gres=gpu:1                 # number of GPUs per node
+#SBATCH --cpus-per-task=10           # number of cores per tasks
+#SBATCH --hint=nomultithread         # we get physical cores not logical
+#SBATCH --distribution=block:block   # we pin the tasks on contiguous cores
+#SBATCH --time=3:00:00              # maximum execution time (HH:MM:SS)
+#SBATCH --output=pytorch_mnist%j.out # output file name
+#SBATCH --error=pytorch_mnist%j.err  # error file name
+#SBATCH --array=1-10
+
+
+cd $WORK/jean-zay-doc/examples/pytorch
+
+module purge
+module load pytorch-gpu/py3/1.4.0 
+GAMMA_STEP=('0.1' '0.2' '0.3' '0.4' '0.5' '0.6' '0.7' '0.8' '0.9' '1.0') 
+srun python ./mnist_example.py --gamma ${GAMMA_STEP[$SLURM_ARRAY_TASK_ID]} &
```

Comment:
Nice it is the first time I see a SLURM job array in real-life! As you probably know I would have done it with `dask-jobqueue` but job arrays is definitely something useful.

Just out of curiosity:
- do you need `srun` if you are running one single task, i.e. could you use:
```
python ./mnist-example.py ...
```
- the `&` + `wait`, is it a common thing (I saw it was used in the Tensorflow examples as well)? Do you need it here since you are running a single task?

Reply:
Well the `&` + `wait` is a relic because I used it when I had multiple `srun`s. With the array or even with the conf file it's not needed.

I don't know about `srun`.

Reply:
I copied the script from the tr example (trying to keep some coherence between the examples) but in fact I think that the `&`, `wait` and `srun` are not necessary for this kind of script. I will remove them before merging.